### ISSUE TITLE
[Performance] Switch NavigationCard to animate with transforms

### ISSUE THREAD

**Issue by ide** [ericvicenti/navigation-rfc#8]
@kmagiera pointed out:

> https://github.com/ericvicenti/navigation-rfc/blob/fc3d05e960239cd4180ee2878e1954ace38a556d/CustomComponents/NavigationCard.js#L104 where we animate right and left props (those require to run layout algorithm each frame and using translateX would be a better choice, not sure if possible to use it though)

**Comment by brentvatne** [comment]
:cool:

**Comment by ericvicenti** [comment]
yeah, using translateX sounds like a better choice for NavigationCard

**Comment by VansonLeung** [comment]
Does translateX support Android too now?

**Comment by sahrens** [comment]
Yes, using transforms would be much better.

@vansonleung: `translateX` should work on Android, but I believe it requires the decomposed form:

```
  style={{
    transform: [
      {translateX: 200},
    ],
  }}
```

**Comment by ericvicenti** [comment]
Had to revert the transform switch because it breaks when debugging in chrome #30
